src/libs/actions/Report.js

```diff
@@ -544,16 +546,23 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
         onyxData.optimisticData[0].onyxMethod = CONST.ONYX.METHOD.SET;
 
         // Also create a report action so that the page isn't endlessly loading
+        const optimisticReportAction = ReportUtils.buildOptimisticCreatedReportAction(newReportObject.ownerEmail);
+        optimisticReportActionID = NumberUtils.rand64();
+        // optimisticReportAction[optimisticReportActionID] = optimisticReportAction[0];
+        // delete optimisticReportAction[0];
+        // console.log('sending optimisticReportActionID', optimisticReportActionID);
+        optimisticReportAction[0].reportActionID = optimisticReportActionID;
```

Comment:
I just checked to see if these will be sorted correctly in App and it does seem like it will from this PR https://github.com/Expensify/App/pull/12626

cc @roryabraham

Reply:
For we need the key to be `0` (aka sequenceNumber). Later we will switch that to key by reportActionID in https://github.com/Expensify/App/pull/13118.

> I just checked to see if these will be sorted correctly in App and it does seem like it will from this PR

An earlier draft of that PR guaranteed that `CREATED` actions would appear first in the sorting order, but it wasn't clear if it was necessary or not so we removed that for simplicity.

Even now, if we're optimistically adding a `CREATED` action when opening a report that doesn't yet exist, I think it's safe to assume that there are no other actions on the report yet and therefore the `CREATED` action will have the lowest `created` timestamp and therefore appear first in the sorting order.

Reply:
Cool so is this PR correct as is? (as long as I remove the commented code which I left to show the other option)

Reply:
Yeah, I think this PR is mostly correct as-is.